docs/building-fcos.md

```diff
@@ -82,6 +82,19 @@ here need to change for that.
 ```sh
 cosa() {
    env | grep COREOS_ASSEMBLER
+   local -r COREOS_ASSEMBLER_CONTAINER_LATEST="quay.io/coreos-assembler/coreos-assembler:latest"
+   if [[ -z ${COREOS_ASSEMBLER_CONTAINER} ]]; then
+       local -r cosa_build_date_str="$(podman inspect -f "{{.Created}}" ${COREOS_ASSEMBLER_CONTAINER_LATEST} | awk '{print $1}')"
+       local -r cosa_build_date="$(date -d ${cosa_build_date_str} +%s)"
+       if [[ $(date +%s) -ge $((cosa_build_date + 60*60*24*7)) ]] ; then
+         echo "----" >&2
+         echo "The COSA container image is more that a week old and likely outdated." >&2
+         echo "You should pull the latest version with:" >&2
+         echo "podman pull ${COREOS_ASSEMBLER_CONTAINER_LATEST}" >&2
+         echo "----" >&2
```

Comment:
totally optional: might be nice to colorize this output to a darkish yellow (mustard) color